extensions/wikia/RTE/RTELinkerHooks.class.php

```diff
@@ -133,4 +133,26 @@ public static function makeBrokenImageLinkObj( $title, $html = '', $query = '',
 		wfProfileOut(__METHOD__);
 		return $ret;
 	}
+
+	/**
+	 * Sets href as # and store proper href in
+	 * data-uncrawlable-url to make redlink
+	 * uncrawlable.
+	 *
+	 * @param $attribs array attributes
+	 *
+	 */
+	public static function onSetHrefForRedLink (&$attribs) {
```

Comment:
Please reformat this method according to https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md

Reply:
Also, RTE extensions isn't a good place for this hook. RTE stands for Rich Text Editor and it's unrelated to redlinks. I think the best place would be to put it in `/extensions/wikia/SEOTweaks` as this is strictly a SEO improvement.